tests/baselines/reference/templateStringInModuleName.errors.txt

```diff
@@ -1,32 +1,32 @@
+tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,1): error TS1433: Unexpected keyword or identifier.
 tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,1): error TS2304: Cannot find name 'declare'.
-tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,9): error TS1005: ';' expected.
 tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,9): error TS2580: Cannot find name 'module'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
-tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,21): error TS1005: ';' expected.
+tests/cases/conformance/es6/templates/templateStringInModuleName.ts(1,16): error TS1442: Template literal not allowed as a string at this position.
+tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,1): error TS1433: Unexpected keyword or identifier.
 tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,1): error TS2304: Cannot find name 'declare'.
-tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,9): error TS1005: ';' expected.
 tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,9): error TS2580: Cannot find name 'module'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
-tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,24): error TS1005: ';' expected.
+tests/cases/conformance/es6/templates/templateStringInModuleName.ts(4,16): error TS1442: Template literal not allowed as a string at this position.
 
 
 ==== tests/cases/conformance/es6/templates/templateStringInModuleName.ts (8 errors) ====
     declare module `M1` {
     ~~~~~~~
+!!! error TS1433: Unexpected keyword or identifier.

```

Comment:
Not a big deal, but this seems redundant/noisy

Reply:
Indeed! I think we can skip these error messages altogether for `declare`d nodes, since the downstream parsing would emit an unhappy diagnostic anyway.

Reply:
The baselines are a bit more verbose but in the CLI we get a much more precise error now:

```tsc
 node ../TypeScript/built/local/tsc          
index.ts:1:16 - error TS1442: Module declaration names may only use ' or " quoted strings.

1 declare module `as23` {}
                 ~~~~~~


Found 1 error.
```